dds_cli/__main__.py

```diff
@@ -1605,6 +1605,11 @@ def get_data(
             "Flag '--get-all' cannot be used together with options '--source'/'--source-path-fail'."
         )
         sys.exit(1)
+    elif not get_all and not (source or source_path_file):
+        LOG.error(
+            "The command 'dds data get' reuires either the flag '--get-all' or some of the options '--source'/'--source-path-fail'."
```

Comment:
Suggestion to change the message a little
```suggestion
            "Specify either '--source'/'--source-path-file' to download specific directories / files, or '--get-all' to download all project contents."
```